collector.go

```diff
@@ -67,6 +74,29 @@ var (
 			"sv_tested":  {GAUGE, "server_testing_connections", 1, "Server connections currently running either server_reset_query or server_check_query, shown as connection"},
 			"sv_login":   {GAUGE, "server_login_connections", 1, "Server connections currently in the process of logging in, shown as connection"},
 			"maxwait":    {GAUGE, "client_maxwait_seconds", 1, "Age of oldest unserved client connection, shown as second"},
+			"pool_mode":  {LABEL, "pool_mode", 1, "The pooling mode in use."},
+		},
+		"lists": {
+			"databases":     {GAUGE, "databases_total", 1, "Number of databases"},
+			"users":         {GAUGE, "users_total", 1, "Number of users"},
+			"pools":         {GAUGE, "pools_total", 1, "Number of pools"},
+			"free_clients":  {GAUGE, "free_clients_total", 1, "Number of free clients"},
+			"used_clients":  {GAUGE, "used_clients_total", 1, "Number of used clients"},
+			"login_clients": {GAUGE, "login_clients_total", 1, "Number of clients in login state"},
```

Comment:
Gauge metrics should not end with `_total` in the name. This is reserved for counters. Please verify if these metrics are gauges or counters and name them appropriately.